azure-maven-plugin-lib/src/main/java/com/microsoft/azure/maven/AbstractAzureMojo.java

```diff
@@ -364,9 +366,11 @@ protected Account getAzureAccount() throws MavenDecryptException, AzureExecution
 
     private Account login(@Nonnull com.microsoft.azure.toolkit.lib.auth.model.AuthConfiguration auth) {
         promptAzureEnvironment(auth.getEnvironment());
+        MavenAuthUtils.disableIdentityLogs();
         accountLogin(auth);
         final Account account = com.microsoft.azure.toolkit.lib.Azure.az(AzureAccount.class).account();
-        if (account.getAuthType() == AuthType.OAUTH2 || account.getAuthType() == AuthType.DEVICE_CODE) {
+        final boolean isInteractiveLogin = account.getAuthType() == AuthType.OAUTH2 || account.getAuthType() == AuthType.DEVICE_CODE;
+        if (isInteractiveLogin) {
```

Comment:
Seems we could remove `if (isInteractiveLogin) {` here as Device code must be interactive